drivers/adc/Kconfig.ads114s0x

```diff
@@ -34,3 +34,10 @@ config ADC_ADS114S0X_GPIO
 
 	  The GPIO functionality is handled by the ADS114S0x GPIO
 	  driver.
+
+config ADC_ADS114S0X_WAIT_FOR_COMPLETION_TIMEOUT_IN_MS
```

Comment:
nit: this should probably be `config ADC_ADS114S0X_WAIT_FOR_COMPLETION_TIMEOUT_MS`, that's how all other time Kconfig entries are defined.

Reply:
done